extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -319,17 +321,43 @@ tags: {  # <amp-story-bookend>
     }
   }
 }
+tags: {  # amp-story-consent (json)
```

Comment:
I am not familiar with the validator rules : ) would https://github.com/ampproject/amphtml/pull/15474/files#diff-2f32ef70db93f072c60fc1a70694c7bbR359 does the job?

Reply:
This rule ensures it's a ``<script>`` with the ``application/json`` attribute, has ``amp-story-consent`` as a parent, etc : )